irods/test/access_test.py

```diff
@@ -332,6 +335,37 @@ def test_ses_acls_data_and_collection_395_396(self):
             self.alice.remove()
             self.team.remove()
 
+    def test_removed_user_does_not_affect_raw_ACL_queries__431(self):
+        user_name = "testuser"
+        session = self.sess
+        try:
+            # create user and collection
+            user = session.users.create(user_name, 'rodsuser')
+            coll_path = "/{0.zone}/home/test".format(session)
+            coll = session.collections.create(coll_path)
+
+            # give user access to collection
+            acces = iRODSAccess('read', coll.path, user.name)
+            session.acls.set(acces)
+
+            # we can get permissions from collection
+            perms = session.acls.get(coll)
```

Comment:
Should we add some assertion(s) about the permissions retrieved here? Saw that Codacy is upset about the unused variable. Same for line 361.

Reply:
yep.